docs/Project.toml

```diff
@@ -1,3 +1,4 @@
 [deps]
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 Soss = "8ce77f84-9b61-11e8-39ff-d17a774bf41c"
+SymPy = "24249f21-da20-56a4-8eb1-6a02cf4ae2e6"
```

Comment:
Should probably also add the `compat` entry here too.

Reply:
Oh, hadn't thought of that. Should there be separate `compat` entries for docs and test? I'd think those fall back to whatever is given for the overall project